internal/testutil/datagen/README.md

```diff
@@ -0,0 +1,37 @@
+# `datagen`
+
+Contains a set of dummy deployments and charts that can be used to scrape static test data from a live cluster, which then developers can add to the `data` folder. This data will be used for integration tests as of https://github.com/newrelic/nri-kubernetes/pull/263
+
+## Requirements
+
+* `kubectl` and `helm` installed and configured
+* Current `kubectl` context pointed to a preferably freshly-created minikube cluster.
+
+## How it works
+
+`datagen.sh` script will deploy a series of dummy and required workloads to the cluster, making it a cluster containing at least one resource of all the resources we can possibly monitor.
+
+More specifically, it will deploy:
+- The KSM chart, for the KSM services/endpoints discovery tests
+- The [`e2e-resources`](../../../e2e/charts/e2e-resources) chart, which includes multiple samples of resources we monitor:
```

Comment:
would make sense to add ksm as a e2e resource? Also Guillermo had issue with that since it is not included by default

Actually the e2e will likely install nri-bundle whenever is available deploying as well the ksm

Reply:
I would have the:
 - scraper chart or even manifest
 - e2e-resources chart including ksm till we have it in nri-bundle

Reply:
If we include KSM in the `e2e-resources` then 2 ksms will be deployed in the `e2e` tests, right? Things should not break but I'd prefer if the e2e scenario could have the most vanilla env possible.

Reply:
re scraper, I thought about adding it to the `e2e-resources` chart hidden behind a value which defautlts to false. WDYT?

Reply:
sounds good! I wanted to avoid if possible to have multiple charts in the repo